src/tools/crossgen2/Common/Compiler/DevirtualizationManager.cs

```diff
@@ -74,6 +74,32 @@ protected virtual MethodDesc ResolveVirtualMethod(MethodDesc declMethod, DefType
             if (declMethod.OwningType.IsInterface)
             {
                 impl = implType.ResolveInterfaceMethodTarget(declMethod);
+
```

Comment:
Could you put this into src\tools\crossgen2\ILCompiler.ReadyToRun\Compiler\DependencyAnalysis\ReadyToRun\DevirtualizationManager.cs instead (right before it calls into `base`)?

Being able to work around like this is ready to run specific.